wildfire/lib/screens/all_updates/all_updates.dart

```diff
@@ -0,0 +1,352 @@
+                              style: GoogleFonts.openSans(
+                                textStyle: TextStyle(
+                                  color: Color(0xff333333),
+                                  fontSize: 13.sp,
+                                ),
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              Column(children: [
+                SizedBox(
+                  height: 16.h,
+                ),
+                SizedBox(
+                  width: 343.w,
+                  height: 21.h,
+                ),
+                SizedBox(height: 8.h),
+                Container(
+                  width: 343.w,
+                  height: 240.h,
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(24.r),
+                    color: Color(0xffebebeb),
+                  ),
+                  child: Column(
+                    children: [
+                      Row(
+                        mainAxisSize: MainAxisSize.min,
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        crossAxisAlignment: CrossAxisAlignment.center,
+                        children: [
+                          StreamBuilder<QuerySnapshot>(
+                              stream: FirebaseFirestore.instance
+                                  .collection('fire-updates')
+                                  .snapshots(),
+                              builder: (BuildContext,
+                                  AsyncSnapshot<QuerySnapshot> querySnapshot) {
+                                if (querySnapshot.hasData) {
+                                  return Center(
+                                    child: CircularProgressIndicator(),
+                                  );
+                                } else
+                                  return ListView.builder(
+                                      itemBuilder: (context, index) {
+                                    querySnapshot.data!.docs
+                                        .map((DocumentSnapshot) {
+                                      return Container(
+                                        width: 343.w,
+                                        height: 240.h,
+                                        decoration: BoxDecoration(
+                                          borderRadius:
+                                              BorderRadius.circular(24.r),
+                                          color: Color(0xffebebeb),
+                                        ),
+                                      );
+                                    });
+                                  });
+                              })
+                        ],
+                      ),
+                      Column(
```

Comment:
This is what you should return inside of you listview and then using the index to get each of the data to the widget needs

Reply:
you used this to replace the container